server/templates/static/homepage.html

```diff
@@ -149,6 +149,27 @@ <h2>{% trans %}Explore the data{% endtrans %}</h2>
         </li>
       </ul>
     </li>
+
+    <li>
+      {# TRANSLATORS: Header for a set of links to the Data Commons Map Explorer tool, where users are able to create maps on many topics. Please see this string in action at https://datacommons.org #}
+      <a class="main" href="{{ url_for('tools.map') }}">{% trans project="Data Commons" %}Create Maps{% endtrans %}</a>:
+      <ul>
+        <li>
+          {# TRANSLATORS: Link to a map showing the median income across counties in the USA. Please see this string in action at https://datacommons.org #}
+          <a class="sub" href="{{ url_for('tools.map') }}#&sv=Median_Income_Person&pc=0&pd=country/USA&pn=United%20States&pt=Country&ept=County">{% trans %}Median Income across US Counties{% endtrans %}</a>,
+        </li>
+        <li>
+          {# TRANSLATORS: Link to a map showing the attainment of Bachelor Degrees or higher across counties in Washington. Please see this string in action at https://datacommons.org #}
+          <a class="sub" href="{{ url_for('tools.map') }}#&sv=Count_Person_EducationalAttainmentBachelorsDegreeOrHigher&pc=1&pd=geoId/53&pn=Washington&pt=State&ept=County">{% trans %}Attainment of Bachelor Degree or Higher across Washington Counties{% endtrans %}</a>,
+        </li>
+        <li>
+          {# TRANSLATORS: Link to a map showing the number of foreign borns across states in USA. Please see this string in action at https://datacommons.org #}
+          <a class="sub" href="{{ url_for('tools.map') }}#%26sv%3Ddc%2Fq18w8j7b2vxm4%26pc%3D1%26pd%3Dcountry%2FUSA%26pn%3DUnited%20States%20of%20America%26pt%3DCountry%26ept%3DState">{% trans %}Foreign Borns across US States{% endtrans %}</a>,
```

Comment:
again, not sure about including foreign born here. how about unemployment rate?

Reply:
done